alfred-telemetry-solr/README.md

```diff
@@ -78,39 +78,41 @@ Following custom metrics have been implemented.
 
 At the moment following metrics are included, for each core tracked by solr:
 
-| Name                                                                   |
-| -----------------------------------------------------------------------|
-| alfresco.nodes{core=<core>,feature="Alfresco Nodes in Index"}          |
-| alfresco.nodes{core=<core>,feature="Alfresco Transactions in Index"}   |
-| alfresco.nodes{core=<core>,feature="Alfresco Stated in Index"}         |
-| alfresco.nodes{core=<core>,feature="Alfresco Unindexed Nodes"}         |
-| alfresco.nodes{core=<core>,feature="Alfresco Error Nodes in Index"}    |
-| alfresco.acls{core=<core>,feature="Alfresco Acls in Index"}            |
-| alfresco.acls{core=<core>,feature="Alfresco Acl Transactions in Index"}|
+| Name                                                       |
+| -----------------------------------------------------------|
+| alfresco.nodes{core=\<core\>,feature="Nodes"}                |
+| alfresco.nodes{core=\<core\>,feature="Unindexed Nodes"}      |
+| alfresco.nodes{core=\<core\>,feature="Error Nodes"}          |
+| alfresco.acls{core=\<core\>,feature="Acls"}                  |
+| alfresco.states{core=\<core\>}                               |
+
+
 
 ### Solr FTS metrics
 
 At the moment following metrics are included, for each core tracked by solr:
 
-| Name                                                                  |
-| ----------------------------------------------------------------------|
-| alfresco.fts{core=<core>,feature="Node count with FTSStatus Clean"}   |
-| alfresco.fts{core=<core>,feature="Node count with FTSStatus Dirty"}   |
-| alfresco.fts{core=<core>,feature="Node count with FTSStatus New"}     |
+| Name                                        |
+| --------------------------------------------|
+| alfresco.fts{core=\<core\>,feature="Clean"}   |
+| alfresco.fts{core=\<core\>,feature="Dirty"}   |
+| alfresco.fts{core=\<core\>,feature="New"}     |
 
 
 ### Solr tracker metrics
 
 At the moment following metrics are included, for each core tracked by solr:
 
-| Name                                                                  |
-| ----------------------------------------------------------------------|
-| alfresco.nodes{core=<core>,feature="Approx transactions remaining"}   |
-| alfresco.nodes{core=<core>,feature="TX lag"}                          |
-| alfresco.nodes{core=<core>,feature="Last Index TX Commit Time"}       |
-| alfresco.acls{core=<core>,feature="Approx change sets remaining"}     |
-| alfresco.acls{core=<core>,feature="Change Set Lag"}                   |
-| alfresco.acls{core=<core>,feature="Last Index Change Set Commit Time"}|
+| Name                                                                            |
+| --------------------------------------------------------------------------------|
+| alfresco.tracker{core=\<core\>,tracker="Nodes"}                                   |    
```

Comment:
I think those all need separate names

Reply:
I agree.
For metrics where you can never do any sensible operations on the group of metrics should be put into separate metrics.

I think I expect something like this: 

`alfresco.tracker.nodes{core=<core>, state="indexed"}` (At least, I think this metric represents indexed nodes?)
`alfresco.tracker.acls{core=<core>,state="indexed"}`
`alfresco.tracker.nodes{core=<core>, state="remaining"}`
`alfresco.tracker.acls{core=<core>, state="remaining"}`
`alfresco.tracker.nodes.lag{core=<core>}`
`alfresco.tracker.acls.lag{core=<core>}`
`alfresco.tracker.nodes.lastIndexCommitTime{core=<core>}`
`alfresco.tracker.acls.lastIndexCommitTime{core=<core>}`

Note: indexed and remaining nodes are in a single metric, since you can calculate the total from them by summing them. It can make sense to do an operation on them.

Reply:
Actually in the tracker metrics we are talking about indexed transactions (for nodes and for acls).
It’s a bit inconsistent semantically, because in the end alfresco_nodes, alfresco_acls, alfresco_fts are also tracker metrics. I will replace the “.tracker” part with “.transactions”.